_episodes/41-code-review.md

```diff
@@ -205,70 +200,66 @@ You will turn this branch into a pull request for your fictional colleague on yo
    Recall that the base branch is where you want your changes to be merged
-   and the compare branch contains your changes.
-5. Click `Create pull request` to open the request
+   and the compare branch contains the changes.
+5. Click `Create pull request` button to open the request.
    ![Creating a new pull request.](../fig/github-create-pull-request.png){: .image-with-shadow width="900px"}
 6. Add a comment describing the nature of the changes,
-   and then submit the pull request by clicking `Create pull request`.
-![Submitting a pull request.](../fig/github-submit-pullrequest.png){: .image-with-shadow width="900px"}
+   and then submit the pull request by clicking `Create pull request` button (in the new window).
+![Submitting a pull request.](../fig/github-submit-pull-request.png){: .image-with-shadow width="900px"}
 7. At this point, the code review process is initiated.
 
 We will now discuss what to look for in a code review,
-before practising this on this fictional change.
+before practising it on this fictional change.
 
-## Reviewing a pull request
+### Reviewing a Pull Request
 
 Once a review has been raised it is over to the reviewer to review the code
-and submit a review.
-
-### Things to look for in a code review
+and submit feedback.
 
 Reviewing code effectively takes practice.
-However, here is some guidance on the kinds of things you should
+However, here is some guidance on what you should
 be looking for when reviewing a piece of code.
 
+#### Things to Look for in a Code Review
+
 Start by understanding what the code _should_ do, by reading the specification/user requirements,
 the pull request description and talking to the developer.
 In this case, understand what [SR1.1.1](../31-software-requirements/index.html#solution-requirements) means.
 
-Once you're happy, start reading the code (skip the test code for now). You're
-going to be assessing the code in 4 key areas:
-
-* Is the code readable
-* Is the code a minimal change
-* Is the structure of the code clear
-* Is there appropriate and up-to-date documentation
+Once you're happy, start reading the code (skip the test code for now - we will come back to it later). 
+You're going to be assessing the code in the following key areas.
 
-**Is the code readable**
+##### Is the proposed code readable?
 
 * Think about do the names of the variables, do they [follow guidelines for good
 names?](../15-coding-conventions/index.html#l#naming-conventions)
 * Do you understand what conditions in each if statements are for?
 * Do the function names match the behavior of the function.
 
-**Is the code a minimal change**
+##### Is the proposed code a minimal change?
 
 * Does the code reimplement anything that already exists, either
 elsewhere in the codebase or in a library you know about?
 * Does the code implement something that isn't on the ticket?
 
-**Is the structure of the code clear**
+##### Is the structure of the proposed code clear?
```

Comment:
```suggestion
##### Is the structure of the code clear?
```

I'd remove 'proposed' here, since you're also looking at the change in the context of the rest of the code as well, i.e. that the rest of the code's structure isn't negatively affected by the proposed change.